src/schema/v2/artwork/artworkConsignmentSubmissionType.ts

```diff
@@ -86,28 +86,84 @@ const ArtworkConsignmentSubmissionType = new GraphQLObjectType<
         description: "Submission state label visible to the user.",
         resolve: ({ state }) => {
           switch (state.toLowerCase()) {
+            case "draft":
+            case "hold":
+            case "closed":
+              return null
             case "approved":
+              return "Approved"
             case "rejected":
-            case "closed":
-            case "published":
-              return "Evaluation Complete"
+              return "Submission Unsuccessful"
             default:
               return "In Progress"
           }
         },
       },
+      stateLabelColor: {
+        type: GraphQLString,
+        resolve: ({ state }) => {
+          switch (state.toLowerCase()) {
+            case "rejected":
+              return "black60"
+            default:
+              return "black100"
+          }
+        },
+      },
+      actionLabel: {
+        type: GraphQLString,
+        description:
+          "Action label asks the user to poseed with the submission.",
+        resolve: ({ state }) => {
+          switch (state.toLowerCase()) {
+            case "draft":
+              return "Complete Submission"
+            case "approved":
+              return "Complete Listing"
+            default:
+              return null
+          }
+        },
+      },
+      buttonLable: {
+        type: GraphQLString,
+        description: "Button label visible to the user.",
+        resolve: ({ state }) => {
+          switch (state.toLowerCase()) {
+            case "draft":
+              return "Complete Submission"
+            case "approved":
+              return "Add Additional Information"
+            case "submitted":
+            case "published":
+              /* case "RESUBMITTED": */
+              return "Edit Submission"
+            default:
+              return null
+          }
+        },
+      },
```

Comment:
Maybe we could define the color for the button like this:

```suggestion
      },
      buttonVariant: {
        type: GraphQLString,
        resolve: ({ state }) => {
          switch (state.toLowerCase()) {
            case "approved":
            case "draft":
              return "primaryBlack"
            default:
              return "secondaryBlack"
          }
        },
      },
```